tests/integration/go/parameterized/sdk/go/go.mod

```diff
@@ -0,0 +1,5 @@
+module example.com/pulumi-pkg/sdk
```

Comment:
This go mod is wrong, it should be "example.com/pulumi-pkg/sdk/go" but https://github.com/pulumi/pulumi/pull/17923 will fix that

Reply:
just tested it and it works, i set the base of this branch to that so the tests can rerun against it as well.